tests/maf/test_stackers.py

```diff
@@ -287,6 +300,35 @@ def test_observation_start_datetime64_stacker(self):
         recovered_mjd = Time(value["observationStartDatetime64"], format="datetime64").mjd
         assert np.allclose(recovered_mjd, data["observationStartMJD"])
 
+    def test_observation_start_timestamp_stacker(self):
+        rng = np.random.default_rng(seed=6563)
+        num_points = 5
+
+        data = pd.DataFrame(
+            {"id": np.arange(num_points), "observationStartMJD": 61000 + 3000 * rng.random(num_points)}
+        )
+
+        stacker = stackers.ObservationStartTimestampStacker("observationStartMJD")
+        value = stacker.run(data)
+        assert value.start_timestamp.dtype == "datetime64[ns, UTC]"
+        assert isinstance(value.start_timestamp[0], pd.Timestamp)
+
+        data = pd.DataFrame(
+            {"id": np.arange(num_points), "observationStartMJD": 61000 + 3000 * rng.random(num_points)}
+        ).to_records(index=False)
+        stacker = stackers.ObservationStartTimestampStacker("observationStartMJD")
+        raw_value = stacker.run(data)
+        value = pd.DataFrame(raw_value)
+        assert value.start_timestamp.dtype == "datetime64[ns, UTC]"
+        assert isinstance(value.start_timestamp[0], pd.Timestamp)
+
+        data = {"id": np.arange(num_points), "observationStartMJD": 61000 + 3000 * rng.random(num_points)}
+        stacker = stackers.ObservationStartTimestampStacker("observationStartMJD")
+        raw_value = stacker.run(data)
+        value = pd.DataFrame(raw_value)
+        assert value.start_timestamp.dtype == "datetime64[ns, UTC]"
+        assert isinstance(value.start_timestamp[0], pd.Timestamp)
```

Comment:
I think this test checks that the dtype of the values is correct, but does not actually test the values of the column. 
Please add a test that a computed value is correct.